src/treebuilders/multiply.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
-#include "mrcpp_declarations.h"
+#include "functions/RepresentableFunction.h"
+#include "trees/FunctionTree.h"
+#include "trees/FunctionTreeVector.h"
```

Comment:
Any reason to include rather than forward declare?

Reply:
I couldn't compile otherwise, but I forgot to try harder after getting it to work.

Reply:
Done, partially. `FunctionTreeVector` is defined as a template alias and it does not look like this construct plays well with forward declarations: https://stackoverflow.com/questions/16304468/forward-declare-a-template-alias